src/main/drivers/dshot.c

```diff
@@ -382,6 +382,10 @@ bool isDshotTelemetryActive(void)
     return false;
 }
 
+bool isDshotTelemetryEnabled(void)
+{
+    return useDshotTelemetry;
```

Comment:
Was hoping to gain some performance using this directly.

Maybe changing this to

```suggestion
    return motorConfig()->dev.useDshotTelemetry;
```

might work more dynamic?

Otherwise happy to close as it only adds an extra layer